### PR TITLE
Homescreen breakpoint update

### PR DESCRIPTION
**Pull request by JGeiser9**

### CODE REVIEW

File: src/components/styles/Card/Card.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ const RightItemDetail = styled.div`
   font-size: 12px;
   margin: 2px 0;
 
-  @media (min-width: 769px) {
+  @media (min-width: 850px) {
```
Comment (shanejann) [inline src/components/styles/Card/Card.js]: So we're keeping these breakpoints, right? Might be worthwhile to call out why they're not one of our `theme` breakpoints (either with a comment, or some other way)

Comment (kilpatty) [reply src/components/styles/Card/Card.js]: Hmm yeah I'm not sure the best way to categorize these. I suppose we could also add in breakpoints that are like sm-md-lg? But open to all ideas here.

Comment (shanejann) [reply src/components/styles/Card/Card.js]: Well 🤔 hmmm... So I don't think a comment is necessary if we use our breakpoint variables everywhere else BUT, how many times do we go 'why is this breaking at _this_ point', ya know? Maybe we need some standard the implies this is an `@exception` or something, ya know?

Comment (kilpatty) [reply src/components/styles/Card/Card.js]: @JGeiser9 Want to just add a comment here that this an exception? I like the syntax Shane used: `//@exception` and put them on both places where we break at 850. Once that's added I think this is good to go.

**Pull request merged**